src/DependencyInjection/Configuration.php

```diff
@@ -624,6 +624,10 @@ private function getHttpDispatcherNode(): ArrayNodeDefinition
                     ->defaultNull()
                     ->info('Httplug async client service name to use for sending the requests.')
                 ->end()
+                ->scalarNode('request_factory')
+                    ->defaultNull()
+                    ->info('Service name of factory for PSR-7 messages.')
```

Comment:
I'm a bit confused right now... is it `PSR-7` or PSR-17`?

The parameter description in `v3` says `PSR-7` https://github.com/FriendsOfSymfony/FOSHttpCache/blob/3.x/src/ProxyClient/HttpProxyClient.php#L43
But the Discovery class is named `PSR-17`: https://github.com/FriendsOfSymfony/FOSHttpCache/blob/3.x/src/ProxyClient/HttpProxyClient.php#L55

Reply:
oh, good catch! i think that has historical reasons: when we started with this, there was no PSR-17 yet and we used some custom factory if i remember correctly. meanwhile, we completely switched to PSR-17, which is the factory for the PSR-7 http messages.